code/_onclick/hud/hud.dm

```diff
@@ -196,6 +196,7 @@
 				screenmob.client.screen -= infodisplay
 
 	for(var/thing in plane_masters)
+		plane_masters[thing].backdrop(screenmob)

```

Comment:
Just noticed this is a 512 secret `:`, is this something we particularly care about avoiding?

Reply:
@tgstation/commit-access

Personally i'd say avoid it.